From ac6b8a0a34f3162523de15676fe4c1095735458a Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Fri, 22 Apr 2016 17:38:24 -0500
Subject: [PATCH 11/23] ARM: dts: support for imx6sxscm flavors

ARM dts files to support the following imx6sxscm flavors for evb board.
    - imx6sxscm 1GB
    - imx6sxscm 512MB
    - imx6sxscm ePoP

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/boot/dts/Makefile                         | 16 +++-
 arch/arm/boot/dts/imx6sxscm-1gb-evb-lcdif1-ldo.dts | 34 ++++++++
 arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-m4.dts     | 98 ----------------------
 arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo.dts        | 12 +--
 arch/arm/boot/dts/imx6sxscm-1gb-evb-m4-ldo.dts     | 96 +++++++++++++++++++++
 arch/arm/boot/dts/imx6sxscm-1gb-evb-mqs-ldo.dts    | 43 ++++++++++
 arch/arm/boot/dts/imx6sxscm-1gb-evb-sai-ldo.dts    | 29 +++++++
 .../boot/dts/imx6sxscm-512mb-evb-lcdif1-ldo.dts    | 10 +++
 arch/arm/boot/dts/imx6sxscm-512mb-evb-ldo.dts      | 10 +++
 arch/arm/boot/dts/imx6sxscm-512mb-evb-m4-ldo.dts   | 17 ++++
 arch/arm/boot/dts/imx6sxscm-512mb-evb-mqs-ldo.dts  | 10 +++
 arch/arm/boot/dts/imx6sxscm-512mb-evb-sai-ldo.dts  | 10 +++
 arch/arm/boot/dts/imx6sxscm-512mb.dtsi             | 15 ++++
 arch/arm/boot/dts/imx6sxscm-emmc.dtsi              | 22 +++++
 .../arm/boot/dts/imx6sxscm-epop-evb-lcdif1-ldo.dts |  9 ++
 arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts       |  9 ++
 arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts    |  9 ++
 arch/arm/boot/dts/imx6sxscm-epop-evb-mqs-ldo.dts   |  9 ++
 arch/arm/boot/dts/imx6sxscm-epop-evb-sai-ldo.dts   |  9 ++
 19 files changed, 359 insertions(+), 108 deletions(-)
 create mode 100644 arch/arm/boot/dts/imx6sxscm-1gb-evb-lcdif1-ldo.dts
 delete mode 100644 arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-m4.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-1gb-evb-m4-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-1gb-evb-mqs-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-1gb-evb-sai-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-512mb-evb-lcdif1-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-512mb-evb-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-512mb-evb-m4-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-512mb-evb-mqs-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-512mb-evb-sai-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-512mb.dtsi
 create mode 100644 arch/arm/boot/dts/imx6sxscm-emmc.dtsi
 create mode 100644 arch/arm/boot/dts/imx6sxscm-epop-evb-lcdif1-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-epop-evb-mqs-ldo.dts
 create mode 100644 arch/arm/boot/dts/imx6sxscm-epop-evb-sai-ldo.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 8144ef6..3901a6f 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -242,9 +242,21 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	vf610-cosmic.dtb \
 	imx6sx-sdb-emmc.dtb \
 	imx6sx-sdb-m4.dtb \
+	imx6sxscm-512mb-evb-ldo.dtb \
+	imx6sxscm-512mb-evb-lcdif1-ldo.dtb \
+	imx6sxscm-512mb-evb-m4-ldo.dtb \
+	imx6sxscm-512mb-evb-mqs-ldo.dtb \
+	imx6sxscm-512mb-evb-sai-ldo.dtb \
 	imx6sxscm-1gb-evb-ldo.dtb \
-	imx6sxscm-1gb-evb-ldo-lcdif1.dtb \
-	imx6sxscm-1gb-evb-ldo-m4.dtb \
+	imx6sxscm-1gb-evb-lcdif1-ldo.dtb \
+	imx6sxscm-1gb-evb-m4-ldo.dtb \
+	imx6sxscm-1gb-evb-mqs-ldo.dtb \
+	imx6sxscm-1gb-evb-sai-ldo.dtb \
+	imx6sxscm-epop-evb-ldo.dtb \
+	imx6sxscm-epop-evb-lcdif1-ldo.dtb \
+	imx6sxscm-epop-evb-m4-ldo.dtb \
+	imx6sxscm-epop-evb-mqs-ldo.dtb \
+	imx6sxscm-epop-evb-sai-ldo.dtb \
 	vf610-twr.dtb \
 	imx7d-12x12-lpddr3-arm2.dtb \
 	imx7d-12x12-ddr3-arm2.dtb \
diff --git a/arch/arm/boot/dts/imx6sxscm-1gb-evb-lcdif1-ldo.dts b/arch/arm/boot/dts/imx6sxscm-1gb-evb-lcdif1-ldo.dts
new file mode 100644
index 0000000..6abb4f8
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-1gb-evb-lcdif1-ldo.dts
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-ldo.dts"
+/ {
+	regulators {
+		reg_lcd_3v3: lcd-3v3 {
+			status = "okay";
+		};
+	};
+
+	sii902x_reset: sii902x-reset {
+		status = "okay";
+	};
+};
+
+&csi1 {
+	status = "disabled";
+};
+
+&lcdif1 {
+	status = "okay";
+};
+
+&i2c1 {
+	sii902x@39 {
+		status = "okay";
+	};
+};
diff --git a/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-m4.dts b/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-m4.dts
deleted file mode 100644
index eadeca9..0000000
--- a/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo-m4.dts
+++ /dev/null
@@ -1,98 +0,0 @@
-/*
- * Copyright (C) 2014 Freescale Semiconductor, Inc.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx6sx-sdb-ldo.dts"
-#include "imx6sxscm-evb.dtsi"
-
-/{
-	memory {
-		linux,usable-memory = <0x80000000 0x1ff00000>,
-				<0xa0000000 0x1ff00000>;
-	};
-};
-
-&iomuxc {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_hog>;
-};
-
-/*
- * The flollowing modules are conflicting with M4, disable them when m4
- * is running.
- */
-&adc1 {
-        status = "disabled";
-};
-
-&adc2 {
-        status = "disabled";
-};
-
-&flexcan1 {
-	status = "disabled";
-};
-
-&flexcan2 {
-	status = "disabled";
-};
-
-&i2c3 {
-        status = "disabled";
-};
-
-&mcctest{
-	status = "okay";
-};
-
-&mcctty{
-	status = "okay";
-};
-
-&ocram {
-	reg = <0x00901000 0xf000>;
-};
-
-&qspi2 {
-	status = "disabled";
-};
-
-&qspi_m4 {
-	status = "okay";
-};
-
-&rpmsg{
-	status = "okay";
-};
-
-&uart2 {
-	status = "disabled";
-};
-
-&clks {
-	fsl,shared-clks-number = <0x23>;
-	fsl,shared-clks-index = <IMX6SX_CLK_PLL2_BUS IMX6SX_CLK_PLL2_PFD0
-			IMX6SX_CLK_PLL2_PFD2 IMX6SX_CLK_PLL3_USB_OTG
-			IMX6SX_CLK_PLL3_PFD1 IMX6SX_CLK_PLL3_PFD2
-			IMX6SX_CLK_PLL3_PFD3 IMX6SX_CLK_PLL4_AUDIO
-			IMX6SX_CLK_PLL5_VIDEO
-			IMX6SX_CLK_OCRAM IMX6SX_CLK_CAN1_SERIAL
-			IMX6SX_CLK_CAN1_IPG IMX6SX_CLK_CAN2_SERIAL
-			IMX6SX_CLK_CAN2_IPG IMX6SX_CLK_CANFD
-			IMX6SX_CLK_ECSPI1 IMX6SX_CLK_ECSPI2
-			IMX6SX_CLK_ECSPI3 IMX6SX_CLK_ECSPI4
-			IMX6SX_CLK_ECSPI5 IMX6SX_CLK_QSPI1
-			IMX6SX_CLK_QSPI2 IMX6SX_CLK_SSI1
-			IMX6SX_CLK_SSI2 IMX6SX_CLK_SSI3
-			IMX6SX_CLK_UART_SERIAL IMX6SX_CLK_UART_IPG
-			IMX6SX_CLK_PERIPH_CLK2_SEL IMX6SX_CLK_OCRAM_ALT_SEL
-			IMX6SX_CLK_I2C1 IMX6SX_CLK_I2C2
-			IMX6SX_CLK_I2C3 IMX6SX_CLK_I2C4
-			IMX6SX_CLK_EPIT1 IMX6SX_CLK_EPIT2>;
-	fsl,shared-mem-addr = <0x91F000>;
-	fsl,shared-mem-size = <0x1000>;
-};
diff --git a/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo.dts b/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo.dts
index 1c50cb1..5d70651 100644
--- a/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo.dts
+++ b/arch/arm/boot/dts/imx6sxscm-1gb-evb-ldo.dts
@@ -1,13 +1,9 @@
 /*
- * Copyright 2012 Freescale Semiconductor, Inc.
- * Copyright 2011 Linaro Ltd.
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
  *
- * The code contained herein is licensed under the GNU General Public
- * License. You may obtain a copy of the GNU General Public License
- * Version 2 or later at the following locations:
- *
- * http://www.opensource.org/licenses/gpl-license.html
- * http://www.gnu.org/copyleft/gpl.html
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
  */
 
 #include "imx6sx-sdb-ldo.dts"
diff --git a/arch/arm/boot/dts/imx6sxscm-1gb-evb-m4-ldo.dts b/arch/arm/boot/dts/imx6sxscm-1gb-evb-m4-ldo.dts
new file mode 100644
index 0000000..ffd8237
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-1gb-evb-m4-ldo.dts
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-ldo.dts"
+/{
+	memory {
+		linux,usable-memory = <0x80000000 0x1ff00000>,
+				<0xa0000000 0x1ff00000>;
+	};
+};
+
+&iomuxc {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_hog>;
+};
+
+/*
+ * The flollowing modules are conflicting with M4, disable them when m4
+ * is running.
+ */
+&adc1 {
+        status = "disabled";
+};
+
+&adc2 {
+        status = "disabled";
+};
+
+&flexcan1 {
+	status = "disabled";
+};
+
+&flexcan2 {
+	status = "disabled";
+};
+
+&i2c3 {
+        status = "disabled";
+};
+
+&mcctest{
+	status = "okay";
+};
+
+&mcctty{
+	status = "okay";
+};
+
+&ocram {
+	reg = <0x00901000 0xf000>;
+};
+
+&qspi2 {
+	status = "disabled";
+};
+
+&qspi_m4 {
+	status = "okay";
+};
+
+&rpmsg{
+	status = "okay";
+};
+
+&uart2 {
+	status = "disabled";
+};
+
+&clks {
+	fsl,shared-clks-number = <0x23>;
+	fsl,shared-clks-index = <IMX6SX_CLK_PLL2_BUS IMX6SX_CLK_PLL2_PFD0
+			IMX6SX_CLK_PLL2_PFD2 IMX6SX_CLK_PLL3_USB_OTG
+			IMX6SX_CLK_PLL3_PFD1 IMX6SX_CLK_PLL3_PFD2
+			IMX6SX_CLK_PLL3_PFD3 IMX6SX_CLK_PLL4_AUDIO
+			IMX6SX_CLK_PLL5_VIDEO
+			IMX6SX_CLK_OCRAM IMX6SX_CLK_CAN1_SERIAL
+			IMX6SX_CLK_CAN1_IPG IMX6SX_CLK_CAN2_SERIAL
+			IMX6SX_CLK_CAN2_IPG IMX6SX_CLK_CANFD
+			IMX6SX_CLK_ECSPI1 IMX6SX_CLK_ECSPI2
+			IMX6SX_CLK_ECSPI3 IMX6SX_CLK_ECSPI4
+			IMX6SX_CLK_ECSPI5 IMX6SX_CLK_QSPI1
+			IMX6SX_CLK_QSPI2 IMX6SX_CLK_SSI1
+			IMX6SX_CLK_SSI2 IMX6SX_CLK_SSI3
+			IMX6SX_CLK_UART_SERIAL IMX6SX_CLK_UART_IPG
+			IMX6SX_CLK_PERIPH_CLK2_SEL IMX6SX_CLK_OCRAM_ALT_SEL
+			IMX6SX_CLK_I2C1 IMX6SX_CLK_I2C2
+			IMX6SX_CLK_I2C3 IMX6SX_CLK_I2C4
+			IMX6SX_CLK_EPIT1 IMX6SX_CLK_EPIT2>;
+	fsl,shared-mem-addr = <0x91F000>;
+	fsl,shared-mem-size = <0x1000>;
+};
diff --git a/arch/arm/boot/dts/imx6sxscm-1gb-evb-mqs-ldo.dts b/arch/arm/boot/dts/imx6sxscm-1gb-evb-mqs-ldo.dts
new file mode 100644
index 0000000..0cfdfd8
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-1gb-evb-mqs-ldo.dts
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-ldo.dts"
+/ {
+	sound-mqs {
+		compatible = "fsl,imx6sx-sdb-mqs",
+				"fsl,imx-audio-mqs";
+		model = "mqs-audio";
+		cpu-dai = <&sai1>;
+		asrc-controller = <&asrc>;
+		audio-codec = <&mqs>;
+	};
+};
+
+&usdhc2 {
+	/* pin conflict with mqs*/
+	status = "disabled";
+};
+
+&mqs {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_mqs>;
+	clocks = <&clks IMX6SX_CLK_SAI1>;
+	clock-names = "mclk";
+	status = "okay";
+};
+
+&sai1 {
+	pinctrl-0 = <>;
+	status = "okay";
+};
+
+&sdma {
+	gpr = <&gpr>;
+	/* SDMA event remap for SAI1 */
+	fsl,sdma-event-remap = <0 15 1>, <0 16 1>;
+};
diff --git a/arch/arm/boot/dts/imx6sxscm-1gb-evb-sai-ldo.dts b/arch/arm/boot/dts/imx6sxscm-1gb-evb-sai-ldo.dts
new file mode 100644
index 0000000..6b6fe8a
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-1gb-evb-sai-ldo.dts
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-ldo.dts"
+/ {
+	sound {
+		cpu-dai = <&sai1>;
+	};
+};
+
+&audmux {
+	/* pin conflict with sai */
+	status = "disabled";
+};
+
+&sai1 {
+	status = "okay";
+};
+
+&sdma {
+	gpr = <&gpr>;
+	/* SDMA event remap for SAI1 */
+	fsl,sdma-event-remap = <0 15 1>, <0 16 1>;
+};
diff --git a/arch/arm/boot/dts/imx6sxscm-512mb-evb-lcdif1-ldo.dts b/arch/arm/boot/dts/imx6sxscm-512mb-evb-lcdif1-ldo.dts
new file mode 100644
index 0000000..2e3780c
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-512mb-evb-lcdif1-ldo.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-lcdif1-ldo.dts"
+#include "imx6sxscm-512mb.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-512mb-evb-ldo.dts b/arch/arm/boot/dts/imx6sxscm-512mb-evb-ldo.dts
new file mode 100644
index 0000000..f34b358
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-512mb-evb-ldo.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-ldo.dts"
+#include "imx6sxscm-512mb.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-512mb-evb-m4-ldo.dts b/arch/arm/boot/dts/imx6sxscm-512mb-evb-m4-ldo.dts
new file mode 100644
index 0000000..69de9a1
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-512mb-evb-m4-ldo.dts
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-m4-ldo.dts"
+/{
+ 	memory {
+		linux,usable-memory = <0x80000000 0x1ff00000> ;
+ 	};
+ };
+ &mcctest{
+	status = "disabled";
+ };
diff --git a/arch/arm/boot/dts/imx6sxscm-512mb-evb-mqs-ldo.dts b/arch/arm/boot/dts/imx6sxscm-512mb-evb-mqs-ldo.dts
new file mode 100644
index 0000000..180305d
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-512mb-evb-mqs-ldo.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-mqs-ldo.dts"
+#include "imx6sxscm-512mb.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-512mb-evb-sai-ldo.dts b/arch/arm/boot/dts/imx6sxscm-512mb-evb-sai-ldo.dts
new file mode 100644
index 0000000..ea5458e
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-512mb-evb-sai-ldo.dts
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6sxscm-1gb-evb-sai-ldo.dts"
+#include "imx6sxscm-512mb.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-512mb.dtsi b/arch/arm/boot/dts/imx6sxscm-512mb.dtsi
new file mode 100644
index 0000000..84915ba
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-512mb.dtsi
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+/ {
+	memory {
+		reg = <0x80000000 0x20000000>;
+	};
+
+};
+
diff --git a/arch/arm/boot/dts/imx6sxscm-emmc.dtsi b/arch/arm/boot/dts/imx6sxscm-emmc.dtsi
new file mode 100644
index 0000000..899ee70
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-emmc.dtsi
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+&usdhc4 {
+	pinctrl-names = "default", "state_100mhz", "state_200mhz";
+	pinctrl-0 = <&pinctrl_usdhc4_1>;
+	pinctrl-1 = <&pinctrl_usdhc4_1_100mhz>;
+	pinctrl-2 = <&pinctrl_usdhc4_1_200mhz>;
+	bus-width = <8>;
+	/*
+	 * overwrite cd-gpios and wp-gpios since they are reused as eMMC DATA
+	 * signals after rework
+	 */
+	cd-gpios = <>;
+	wp-gpios = <>;
+	non-removable;
+	status = "okay";
+};
diff --git a/arch/arm/boot/dts/imx6sxscm-epop-evb-lcdif1-ldo.dts b/arch/arm/boot/dts/imx6sxscm-epop-evb-lcdif1-ldo.dts
new file mode 100644
index 0000000..2d0dc78
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-epop-evb-lcdif1-ldo.dts
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#include "imx6sxscm-512mb-evb-lcdif1-ldo.dts"
+#include "imx6sxscm-emmc.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts b/arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts
new file mode 100644
index 0000000..62bc1ff
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-epop-evb-ldo.dts
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#include "imx6sxscm-512mb-evb-ldo.dts"
+#include "imx6sxscm-emmc.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts b/arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts
new file mode 100644
index 0000000..ab60602
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-epop-evb-m4-ldo.dts
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#include "imx6sxscm-512mb-evb-m4-ldo.dts"
+#include "imx6sxscm-emmc.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-epop-evb-mqs-ldo.dts b/arch/arm/boot/dts/imx6sxscm-epop-evb-mqs-ldo.dts
new file mode 100644
index 0000000..d0e75e6
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-epop-evb-mqs-ldo.dts
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#include "imx6sxscm-512mb-evb-mqs-ldo.dts"
+#include "imx6sxscm-emmc.dtsi"
diff --git a/arch/arm/boot/dts/imx6sxscm-epop-evb-sai-ldo.dts b/arch/arm/boot/dts/imx6sxscm-epop-evb-sai-ldo.dts
new file mode 100644
index 0000000..0def540
--- /dev/null
+++ b/arch/arm/boot/dts/imx6sxscm-epop-evb-sai-ldo.dts
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+#include "imx6sxscm-512mb-evb-sai-ldo.dts"
+#include "imx6sxscm-emmc.dtsi"
-- 
1.9.1

